cranelift/codegen/src/ir/entities.rs

```diff
@@ -88,12 +88,15 @@ impl Value {
 ///
 /// Most usage of `Inst` is internal. `Inst`ructions are returned by
 /// [`InstBuilder`](super::InstBuilder) instructions that do not return a
-/// [`Value`], such as control flow and trap instructions.
+/// [`Value`], such as control flow and trap instructions, as well as instructions that return a
+/// variable (potentially zero!) number of values, like call or call-indirect instructions.
 ///
 /// If you look around the API, you can find many inventive uses for `Inst`,
 /// such as [annotating specific instructions with a comment][inst_comment]
 /// or [performing reflection at compile time](super::DataFlowGraph::analyze_branch)
-/// on the type of instruction.
+/// on the type of instruction. A more typical use of `Inst` would be to obtain the results of a
+/// call via [`inst_results`](super::DataFlowGraph::inst_results) or its analogue in
+/// `cranelift_frontend::FuncBuilder`.
```

Comment:
i'd be inclined to move this to the front of the paragraph, and leave inventive uses as "and also, here are more exotic uses", but i really don't know how often people are doing `builder.ins().call()` that they might end up here!